src/BenchmarksDriver/Program.cs

```diff
@@ -1082,11 +1082,24 @@ public static int Main(string[] args)
 
                 if (monoOption.HasValue() && !serverJob.EnvironmentVariables.ContainsKey("MONO_ENV_OPTIONS"))
                 {
+                    string mono_runtime_value = monoOption.Value();
```

Comment:
Please follow the naming conventions already used:
`monoRuntimeValue` for instance (camel case)